3rdPartyCypress/cypress/integration/mediaAssetPagePrerollAds.js

```diff
@@ -0,0 +1,87 @@
+import getAppEnv from '../../../cypress/support/helpers/getAppEnv';
+import appToggles from '../../../cypress/support/helpers/useAppToggles';
+import { getEmbedUrl } from '../../../cypress/integration/pages/mediaAssetPage/helpers';
+
+const environment = getAppEnv();
+
+describe('Media Asset Pages - Preroll Ads', () => {
+  describe('should not load the preroll ad plugin', () => {
+    const mapsWithoutPrerollConfig = {
```

Comment:
Should these config objects be housed in another file & imported, as is the pattern in other bits of Cypress?

Reply:
Can do, thanks for the suggestion.